demo/docs/tutorials/Custom Inputtype.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: Creating a custom InputType
+id: custom-inputtype
+slug: /custom-inputtype
```

Comment:
I'd recommend using custom-input-type instead, same for the file name(Custom Input Type)